File: templates/python/fluid_cfd_state_map.py
"""fluid_cfd_state_map: 流体与 CFD状态热力图（heatmap 模式，合成数据）."""
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', '_utils', 'python'))
from generated_patterns import make_template_figure


def make_figure(title='fluid and CFD analysis: state heatmap'):
    return make_template_figure('heatmap', seed=2603, title=title, domain='fluid and CFD analysis', topic='state heatmap')


if __name__ == '__main__':
    fig = make_figure()
    fig.savefig(__file__.replace('.py', '.png'), dpi=150)

File: templates/matlab/fluid_cfd_state_map.m
function fig = fluid_cfd_state_map()
    addpath(fullfile(fileparts(mfilename('fullpath')),'..','..','_utils','matlab'));
    fig = generated_template_figure('heatmap', 2603, 'fluid and CFD analysis: state heatmap', 'fluid and CFD analysis', 'state heatmap');
end
